src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  max-width: 1200px;
+  margin: 0 auto;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: $header-shadow;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__nav-list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__nav-item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__nav-link {
+    position: relative;
+    display: block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $title-color;
+    transition: text-transform 0.3s ease;
+  }
+
+  &__nav-link:hover,
+  .is-active {
+    color: $button-color;
+    transition: all 0.3s ease;
```

Comment:
it's not a good practice to use default value `all` as transition-property. Specify what exactly you'd like to change